airflow/providers/http/sensors/http.py

```diff
@@ -62,6 +64,9 @@ def response_check(response, task_instance):
     :param endpoint: The relative part of the full url
     :param request_params: The parameters to be added to the GET url
     :param headers: The HTTP headers to be added to the GET request
+    :param response_error_codes_whitelist: An whitelist to return False on poke(), not to raise exception.
```

Comment:
please change variable name from `response_error_codes_whitelist` to `response_error_codes_allowlist`

Reply:
Thanks..! I'll be more careful in naming variable..! :bow: